src/utils/hdfs_mods.py

```diff
@@ -141,3 +150,34 @@ def check_file_exists(filename: str, filepath: str = "./data/raw/") -> bool:
         raise FileNotFoundError(f"File {filename} does not exist or is empty")
 
     return output
+
+
+def hdfs_mkdir(path):
+    """Function to create a directory in HDFS
+
+    Args:
+        path (string) -- The path to create
+    """
+    hdfs.mkdir(path)
+    return None
+
+
+def hdfs_open(filepath, mode):
```

Comment:
These new hdfs functions are missing unit test mocks. It can only be done on DevTest

Reply:
I have made a ticket for this